<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Inspector</class>
 <widget class="QWidget" name="Inspector" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Inspector</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item>
    <widget class="QToolButton" name="buttonBack" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="toolTip" >
      <string/>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../datamodel/icons.qrc" >:/icons/icons/undo.png</iconset>
     </property>
     <property name="shortcut" >
      <string>Backspace</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget" />
     <widget class="QTableWidget" name="tableWidget" >
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../datamodel/icons.qrc" />
 </resources>
 <connections/>
</ui>
